src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -65,6 +66,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case DeleteCommand.COMMAND_WORD:
             return new DeleteCommandParser().parse(arguments);
 
+        case DeleteTaskCommand.COMMAND_WORD:
+            return new DeleteTaskCommandParser().parse(arguments);
```

Comment:
Would it be good to add a test case for DeleteTaskCommand under AddressBookParserTest?